Finished COVID form

Hackinet opened the pull request:
<!--
(Thanks for sending a pull request!
-->

<!-- Add the issue number that is fixed by this PR (In the form Fixes #123) -->
Fixes  #3241

#### Short description of what this resolves:

I worked on the COVID Form. I had a previous PR which I closed(because I updated the code and hence this new PR).

#### Changes proposed in this pull request:
- I finished the upper half of the COVID form, 'Basic Information'.
- I also added new variables(array choices) in the file new.php that would be used in the twig.
- Those variable names were taken from here:https://www.cdc.gov/coronavirus/2019-ncov/downloads/data-dictionary.pdf

robertdown commented:
Related to #3241

robertdown commented:
Can you post a couple of screen shots of the form? I'm looking at it locally and I'm seeing a lot of spacing issues. Maybe it's my install? Some screenshots will help me compare and figure out what's going on

robertdown commented on contrib/forms/covid19/cdc-pui-case-report-form/templates/form.html.twig:
```diff
@@ -80,20 +84,270 @@
+               <div class="col-4">
+                   <div class="form-group row">
+                        <label for="hc_work_yn" class="col-7 col-form-label pt-1 text-right">Is the patient a health care worker in the United States?</label>
+                        <select name="hc_work_yn" id="hc_work_yn" class="col-5 form-control">
```
This select box didn't get converted to a select2 for some reason

Hackinet replied:
Just checked, that's because I repeated the id 'hc_work_yn'. :(
![image](https://user-images.githubusercontent.com/11132005/77825960-90ced900-7132-11ea-93a7-5d4ef500e777.png)

robertdown commented:
Will need to wrap all the text and translate it. So instead of `Current Status` it should be `{{ "Current Status"|xl }}`

The `xl` twig filter is just a pass through to our own internal `xl()`

That's the one we should be using, right @bradymiller ?

Hackinet commented:
> Can you post a couple of screen shots of the form? I'm looking at it locally and I'm seeing a lot of spacing issues. Maybe it's my install? Some screenshots will help me compare and figure out what's going on
![screencapture-openemr_covid_form_screenshot](https://user-images.githubusercontent.com/11132005/77825992-bc51c380-7132-11ea-89ee-3d50c284dd6c.png)

Hackinet commented:
Hi! @robertdown @bradymiller I added a new commit. 
**Updates in new commit:** 
- Finished The Form 
- Fixed Spacing Issue
**Screenshot:**
![covid_form_full_openemr](https://user-images.githubusercontent.com/11132005/77836632-dcf33b00-717d-11ea-9d4d-9e50001df90f.png)

bradymiller commented:
@robertdown , twig is not supported in 5.0.2, so not sure how these can be ported back. If goal is to support 5.0.2, then would consider not using twig.(untwigging is straightforward, so wouldn't be that big of deal)

bradymiller commented:
for twig, here's the list of translating/escaping functions that can be used (basically the same set as what is currently used):
https://github.com/openemr/openemr/blob/master/src/Core/TwigExtension.php

stephenwaite commented:
@juggernautsei 's patch for weno eRx on `5.0.2.1` adds twig with the following but also had to change ownership to root and writable for `composer.lock` and `composer.json` in the docker
```
composer require twig/twig:^2.0

composer require twig/extensions
```
and has an addition to `interface/globals.php` at line 259
```
// Load twig support
$twigLoader = new Twig\Loader\FilesystemLoader($webserver_root.'/templates');
$twigEnv = new Twig\Environment($twigLoader, ['autoescape' => false]);
$twigEnv->addExtension(new OpenEMR\Core\TwigExtension());
$GLOBALS['twig'] = $twigEnv;
```
not sure why this is changed to nolog at line 281
```
$glrow = sqlQueryNoLog("SHOW TABLES LIKE 'globals'");
```

robertdown commented:
I'll work on reviewing this. I'd hate to un-twig everything, but I guess for portability we probably should. I'll work on that

stephenwaite commented:
we'll have patch 2 coming out soon hopefully but think remember @bradymiller stating that it would be difficult to have composer be part of a patch

robertdown commented:
Bringing this in but we need to address some styling issues. This will be brought into the special COVID branch, not into master. Will go into master once it's functional
